spec/requests/v1/mixtape_request_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe 'mixtape endpoints' do
+  describe 'POST #create' do
+    it 'returns JSON for a single mixtape' do
+      artist = create(:artist)
+      songs = create_list(:song, 5, artist: artist)
+
+      post(mixtapes_url, {no_song: 4,artist: artist}.to_json , accept_headers)
```

Comment:
Spaces after commas.

Have you been able to setup the Rubocop helper in Sublime? It'll help you catch this small little style items before a PR.

I can help set it up tomorrow if you want.